shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -25,6 +25,19 @@ const serverStateToProps = (notifications: Types.NotificationsState, type: 'desk
   return 'never'
 }
 
+type StateProps = {
+  // Set only when the state has a selected conversation, inbox, and
+  // notifications object.
+  _props?: {
+    channelWide: boolean,
+    conversationIDKey: string,
+    desktop: Types.NotifyType,
+    mobile: Types.NotifyType,
+    muted: boolean,
+    saveState: Types.NotificationSaveState,
+  },
+}
+
 const mapStateToProps = (state: TypedState): StateProps => {
   const conversationIDKey = Constants.getSelectedConversation(state)
   if (!conversationIDKey) {
```

Comment:
and then `return {}` => `return null`

Reply:
and yes, you can't return null here, always an object